library/Java/Lang/String/String.hpp

```diff
@@ -934,7 +934,7 @@ namespace Java {
              * @param stringValue
              * @return a String containing stringValue.
              */
-            static String valueOf(const String stringValue);

```

Comment:
I think this is a wrong fix. it should be `const String &stringValue)`

Reply:
Fixed

Reply:
LGTM